cns/restserver/ipam.go

```diff
@@ -47,9 +47,27 @@ func (service *HTTPRestService) requestIPConfigHandlerHelper(ctx context.Context
 		}, errors.New("failed to validate ip config request")
 	}
 
+	var podBackendInfo cns.PodIpInfo
+	if ipconfigsRequest.BackendInterfaceMacAddress != "" {
+		PnPID, err := service.getPNPIDFromMacAddress(ipconfigsRequest.BackendInterfaceMacAddress)
+		if err != nil {
+			return &cns.IPConfigsResponse{
+				Response: cns.Response{
+					ReturnCode: types.FailedToAllocateIPConfig,
```

Comment:
Is this the right error message to return?

Reply:
I will add a new error code.

Reply:
Please also update the message below as well once you change the error code

Reply:
we are technically failing to allocate IP Config due to `getPNPIDFromMacAddress` failed 🙂, so i think we can keep this and just change the error message below